### PR TITLE
Upstash Redis Dashboard Rework

### PR DESCRIPTION
**Pull request by fahreddinozcan**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by emarsha94 — APPROVED** [review]

**Comment by emarsha94** [comment]
@fahreddinozcan Is this a feature request for Redis to update their dashboard?

**Comment by fahreddinozcan** [comment]
Hey,

we've internally updated the structure of the Upstash dashboard and added few more charts etc. This is not finalized yet, but I'll ask for your review when it's finalized.

**Comment by fahreddinozcan** [comment]
Hey @JoshPatel13, this is finalized now. Open for review

File: upstash/metadata.csv
Comment (emarsha94) [inline upstash/metadata.csv:1]: It looks like you are adding metrics to the integration. If so, you will need to update the changelog to reflect that new metrics were added to the integration

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: upstash/metadata.csv
Comment (emarsha94) [reply upstash/metadata.csv:1]: @fahreddinozcan please see above

Comment (fahreddinozcan) [reply upstash/metadata.csv:1]: Hey, thanks for the heads up. Fix is on the way!

Comment (fahreddinozcan) [reply upstash/metadata.csv:1]: @emarsha94 Did the relevant update, fyi

**Review by emarsha94 — APPROVED** [review]

**Comment by buggyhunter** [comment]
@emarsha94 @JoshPatel13 Can we merge the PR if all is OK?

File: upstash/assets/dashboards/upstash_redis_overview.json [hunk 1]
```diff
@@ -0,0 +1,1008 @@
+{
+  "title": "Upstash Redis - Overview",
```
Comment (JoshPatel13) [inline upstash/assets/dashboards/upstash_redis_overview.json:2]: This new dashboard needs to be added to the `manifest.json`. That is why the validations are failing :)

**Review by JoshPatel13 — CHANGES_REQUESTED** [review]

**Comment by fahreddinozcan** [comment]
@emarsha94 @JoshPatel13 I guess it's now ready?

**Review by emarsha94 — APPROVED** [review]

**Comment by bgoldberg122** [comment]
Hi 👋 I just want to confirm that you want to delete the original dashboard rather than keep both. Once resolved I can merge

**Comment by fahreddinozcan** [comment]
@bgoldberg122 We wanted to replace the old one. We won't maintain the old one.

**Pull request merged**